rust/actix/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "server"
+version = "0.1.0"
+authors = ["Oisín Mac Fhearaí <denpashogai@gmail.com>"]
+edition = "2018"
+
+[dependencies]
+actix-web = "2.0.0"
+actix-rt = "1.0.0"
```

Comment:
Please use `*`. Specifying the version of `actix-rt` is the job of `actix-web` in https://github.com/actix/actix-web/blob/206733188478f4d18df376a72fac3a4938b99bd6/Cargo.toml#L66

Reply:
Hm, this specification is common usage and I think it's fine.

Reply:
@JohnTitor but if `actix-rt` version is specified `here`, it requires to update it on each `actix-web` update. Is it not more flexible to use `*` as `actix-web` specify the version of `actix-rt` ?